changelog/fragments/k8s-1.24-bump.yaml

```diff
@@ -0,0 +1,122 @@
+
+        - Replace `k8s.io/api v0.23.5` with `k8s.io/api v0.24.0`
+        - Replace `k8s.io/apimachinery v0.23.5` with  `k8s.io/apimachinery v0.24.0`
+        - Replace `k8s.io/client-go v0.23.5` with `k8s.io/client-go v0.24.0`
+        - Replace `sigs.k8s.io/controller-runtime v0.11.2` with `sigs.k8s.io/controller-runtime v0.12.1`
+        - Hybrid Helm (hybrid) only: Replace `github.com/operator-framework/helm-operator-plugins v0.0.11` with `github.com/operator-framework/helm-operator-plugins v0.0.12-0.20220608155702-d3967d2ae2ac`
+
+        2. Run `go mod tidy` to ensure all dependencies are upgraded
+        
+        3. In `Makefile`:
+        - Replace `CONTROLLER_TOOLS_VERSION ?= 0.8.0` with `CONTROLLER_TOOLS_VERSION ?= 0.9.0`
+        - Replace `ENVTEST_K8S_VERSION = 1.23` with `ENVTEST_K8S_VERSION = 1.24`
+
+        4. Run `make` to run all make targets
+
+  - description: >
+      For Go (go/v3) and Hybrid Helm (hybrid.helm/v1-alpha) language based operators: Add support for Go 1.18
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Go (go/v3) and Hybrid Helm (hybrid.helm/v1-alpha) language based operators - Add support for Go 1.18
+      body: |
+        To update your project to use Go 1.18:
+
+        1. In the `go.mod` file:
+
+        - Replace `go 1.17` with `go 1.18`   
+
+        2. Run `go mod tidy` to ensure all dependencies are upgraded
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): fix channel naming validation [More info](https://github.com/operator-framework/api/pull/231)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "bugfix"
+
+    # Is this a breaking change?
+    breaking: false
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): Add CRD description check validation [More info](https://github.com/operator-framework/api/pull/234)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+  - description: >
+      For `operator-sdk bundle validate` command and GoodPractices validator (`--select-optional name=good-practices`): Add a new check to warn authors when permissions to create CRDs are found [More info](https://github.com/operator-framework/api/pull/241)
+      
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
```

Comment:
We can also add the Hibridy Helm to check in the changelog

```
  - description: >
      For Helm Hybrid (hybrid.helm/v1-alpha) based-language operator: Upgrade dependence from `v0.0.10` to `v0.0.11`. [More info](https://github.com/operator-framework/helm-operator-plugins/releases)
    breaking: false   
```

Reply:
commit https://github.com/operator-framework/operator-sdk/pull/5843/commits/1ccecca4e08393c453999d188a1bc92e3a3fa549